blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/io/IntegrationTestBase.java

```diff
@@ -129,6 +129,26 @@ public void tearDown() throws Exception {
         // meh
     }
 
+    /**
+     * create a number of test locators of numTenants x numMetricNames combination
+     * @param numTenants
+     * @param numMetricNames
+     * @return list of locators for testing
+     */
+    protected List<Locator> generateTestLocators(int numTenants, int numMetricNames) {
+        String tenantId = "100000";
+        String metricName = "io.integration.test";
+
+        List<Locator> locators = new ArrayList<Locator>();
+        for (int i = 1; i <= numTenants; i++) {
+            for (int j = 1; j <= numMetricNames; j++) {
+                locators.add(Locator.createLocatorFromPathComponents(tenantId + i, metricName + j));
```

Comment:
@ChandraAddala likes it if you put the test class name somewhere in the metricName so it's easier to tell which test generates which metrics.